### PR TITLE
For F3R and Denis' review.

### PR DESCRIPTION
**Pull request by charkcl**
SoundJS seems to be not working on local, using BUZZ instead. Disabled Pikachu: responsive To be added: Timer and sound when box is removed. Need to clean code once confirm sounds.js is not needed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,102 +11,105 @@
...
+          <h2 class="modal-title">Instructions for Bishi Bashi Special</h2>
+        </div>
+
+        <div class="modal-body">
+          <img class="img-responsive-height center-block" src="images/PikachuLeft2.png"  alt="Image">
+        </div>
+
+        <div class="modal-footer text-info">
+          <div class="row">
+            <div class="col-xs-10 h5">Recreated by GA-WDI-HK-9-Charlotte KWAN
```
Comment (f3r) [inline index.html]: :)

``` html
<img src="https://ga-dash.s3.amazonaws.com/production/assets/logo-9f88ae6c9c3871690e33280fcf557f33.png">
```

File: js/application.js [hunk 2]
```diff
@@ -1,9 +1,31 @@
 $(document).ready(function(){
 
+//Buzz -register sounds and error alert
+  var bgmusic = new buzz.sound("./sounds/juicy_breakout-theme.mp3");
+  var kickBox = new buzz.sound("./sounds/ball-wall.mp3");
+  var shakeSound = new buzz.sound("./sounds/pling2.mp3");
+  var combo = new buzz.sound("./sounds/ball-paddle.mp3");
+  var pika = new buzz.sound("./sounds/pikapika.wav");
+  if (!buzz.isMP3Supported()) {
+    alert ("Your browser doesn't support MP3 Format.")
```
Comment (f3r) [inline js/application.js:10]: `alert` is a bit aggresive but ok :)

File: js/application.js [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
   //Initialize box colors
-  var initializeBlocks = function(column){
+  var initializeBlocks = function(column, offset){
+    var html = '<div class="blocks col-xs-5 col-xs-offset-' + offset + '">&nbsp</div>';
+    for (var i = 0; i < 20; i++){
+      $(column.columnID).append(html);
+    }
     column.sequence.forEach(function(color, idx){
-      $(column.columnID + " div").eq(-(idx+1)).css("background-color",color);
+      $(column.columnID + " div").eq(-(idx+1)).addClass(color);
```
Comment (f3r) [inline js/application.js:28]: good!

File: js/application.js [hunk 4]
```diff
@@ -41,35 +74,86 @@ $(document).ready(function(){
...
+    if (keycode <100) { return game.playerA; }
     else              { return game.playerB; }
   };
 
-  //Resizing Pikachus
-  $(window).on("resize", function () {
-    var pikaHeight1 = $('#Pikachu1').width() * 1.44;
-    $('#Pikachu1').height(pikaHeight1+"");
-  });
+// STOPWATCH
```
Comment (f3r) [inline js/application.js]: this you planning on working on it later? if not pls remove

File: js/application.js [hunk 5]
```diff
@@ -41,35 +74,86 @@ $(document).ready(function(){
...
 
-  game = new Game();
+  var init = function() {
+    game = new Game();
+    initializeBlocks(game.columnA, 5);
+    initializeBlocks(game.columnB, 2);
+  };
+
+  var reset = function() {
+    console.log("test");
```
Comment (f3r) [inline js/application.js]: remove trivial console.logs

File: js/application.js [hunk 6]
```diff
@@ -41,35 +74,86 @@ $(document).ready(function(){
...
+  // }
+
+  // function timerStart(){
+  //   if (game.playerA.playerScore + game.playerB.playerScore == 1){
+  //     timer();
+  //     console.log("timer working")
+  //   }
+  // }
 
   ////////////////////////////////////////////////////////////////////
```
Comment (f3r) [inline js/application.js:84]: `HELP!! myModal Application.js line#119 not working..:( #3`

Line 119?

File: js/application.js (hunk 5 above)
Comment (charkcl) [inline js/application.js]: Done. Thanks

File: js/application.js (hunk 3 above)
Comment (charkcl) [inline js/application.js:28]: :) Thanks to good teachers.

File: js/application.js (hunk 2 above)
Comment (charkcl) [inline js/application.js:10]: :) of course when music doesn't work!

**Pull request merged**